src/helpers.rs

```diff
@@ -71,7 +71,7 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
     /// Get the `Place` for a local
     fn local_place(&mut self, local: mir::Local) -> InterpResult<'tcx, PlaceTy<'tcx, Tag>> {
         let this = self.eval_context_mut();
-        let place = mir::Place { base: mir::PlaceBase::Local(local), projection: Box::new([]) };
+        let place = mir::Place { base: mir::PlaceBase::Local(local), projection: None };
```

Comment:
You need to update your nightly

Reply:
yeah sorry, I forgot about that and added the file by accident